default.json5

```diff
@@ -22,11 +22,11 @@
       labels: ["dev-dependencies"],
     },
     {
-      matchLanguages: ["python"],
+      matchCategories: ["python"],
```

Comment:
"category" is renamed to "language" across Renovate, as it has a broader meaning and not everything is a language.